rest/src/main/kotlin/service/RestService.kt

```diff
@@ -12,19 +12,17 @@ import kotlin.contracts.contract
 public abstract class RestService(@PublishedApi internal val requestHandler: RequestHandler) {
 
     @PublishedApi
-    internal suspend inline fun <T> call(route: Route<T>, builder: RequestBuilder<T>.() -> Unit = {}): T {
+    internal suspend inline fun <T> call(
+        route: Route<T>,
+        baseUrl: String = Route.baseUrl,
+        builder: RequestBuilder<T>.() -> Unit = {},
+    ): T {
```

Comment:
```suggestion
   internal suspend inline fun <T> call(route: Route<T>, builder: RequestBuilder<T>.() -> Unit = {}): T {
```
we don't change the base url internally, this can be done with `RequestHandler.intercept`